requirements.txt

```diff
@@ -1,7 +1,8 @@
+numpy==1.16.1
```

Comment:
Maybe we shall consider just a `.` in this file, to avoid duplication:

https://stackoverflow.com/questions/14399534/reference-requirements-txt-for-the-install-requires-kwarg-in-setuptools-setup-py